### PR TITLE
(PUP-4067) Create

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

**Comment by haus** [comment]
:+1: Love the commit message especially

**Comment by joshcooper** [comment]
:+1:

File: wix/puppet.wxs [hunk 1]
```diff
@@ -357,6 +357,11 @@
                 <CreateFolder />
               </Component>
             </Directory>
+            <Directory Id="CodeModulesDir" Name="modules">
+              <Component Id="CodeModulesDir" Permanent="yes" Guid="ae4a85a4-a305-4c95-b254-c2f06c586482">
+                <CreateFolder />
+              </component>
```
Comment (Iristyle) [inline wix/puppet.wxs]: FYI there was a typo here @melissa that I corrected in the commit that I merged.

Since it's XML, case sensitivity matters.

Comment (melissa) [inline wix/puppet.wxs]: oops... thanks for catching that!

**Pull request closed without merging**